index.js

```diff
@@ -68,6 +68,26 @@ setup(app, {
 	onInitModel,
 	onInitHooks,
 })
+.then(result => {
+	const email = process.env.BALENA_ROOT_EMAIL || '';
+	const password = process.env.BALENA_ROOT_PASSWORD || '';
+
+	if (email != '' && password != '') {
+		console.log("Creating root account...")
+		const { runInTransaction } = require('./src/platform');
+		const { registerUser } = require('./src/platform/auth');
+
+		return runInTransaction(tx =>
+			registerUser({ username: 'root', email, password }, tx),
+		)
+		.catch(() => {
+			console.log("Root account already exists!")
+		})
+		.return(result);
+	} else {
+		return result;
+	}	
+})
```

Comment:
Can you please move this into its own function and just invoke from here?

Reply:
sure thing

Reply:
You can invoke it as `.tap(onInitCreateAdmin)`